### DISCUSSION TITLE
CDS Units - Feet, Meters, Both

### DISCUSSION THREAD

**Discussion by schnuerle** [openmobilityfoundation/curb-data-specification#15] [Data Specification]
What unit of measurement should be used throughout CDS? Currently we specify meters as a decimal (eg, 6 or 10.5). 

But are cities using and thinking in feet in some areas and is it a chore to convert integer feet measurements to meters everywhere (eg. 6 feet may have to be 1.8288 meters)? If we need feet, where do we specify the unit, next to each field occurrence (eg, ‘length’ and ‘length_units’), or more globally at a higher level in the spec?

**Comment by jiffyclub** [comment]
To throw out something completely different, I'd love to use something like centimeters so that all lengths can be expressed as integers, kind of the same logic behind using millisecond timestamps.

**Reply by schnuerle** [reply]
That's not a bad idea, to allow integers and make it small enough to be converted between feet and meters in a way that makes some kind of sense. Eg, converting 6 feet into 1.83 meters, or 2 meters into 6.56 feet seems clunky but turning 6 feet into 183 cm and 2 meters into 200cm seems cleaner.
